src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Dropdown.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public class Dropdown
+    {
+        /// <summary>
+        /// Represents the label shown for dropdown selector
+        /// </summary>
+        public string Label;
+
+        /// <summary>
+        /// Tuple representing Data. 
+        /// First Item represents item in Dropdown, Second Item represents whether selected by default in dropdwon, Third Item represents the body after dropdown selection
+        /// </summary>
+        public List<Tuple<string, bool, Response>> Data;
+
+        /// <summary>
+        /// Creates an instance of Dropdown Class
+        /// </summary>
+        /// <param name="label">Dropdown Label</param>
+        /// <param name="data">Dropdown Data</param>
+        public Dropdown(string label, List<Tuple<string, bool, Response>> data)
```

Comment:
What would happen if multiple are selected? Might be a question for the UI

Reply:
the last one gets selected :)